apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/ContentTypePreviewPathSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Heading, Paragraph } from '@contentful/f36-components';
+import { Dispatch, SetStateAction } from 'react';
+
+import { ContentTypePreviewPathSelectionList } from './ContentTypePreviewPathSelectionList/ContentTypePreviewPathSelectionList';
+import { styles } from '../ConfigScreen.styles';
+import { AppInstallationParameters } from '../../../types';
+
+interface Props {
+  parameters: AppInstallationParameters;
+  dispatchParameters: Dispatch<SetStateAction<any>>;
+}
+
+export const ContentTypePreviewPathSection = ({ parameters, dispatchParameters }: Props) => {
+  // TO DO: Adjust logic to limit content type duplication of preview path and token
```

Comment:
notes in the PR description, follow-up work to determine if we want to support multiple `previewPaths` per `contentTypes` selected. At this stage, we do not have logic to prevent that.